build/v1/types.go

```diff
@@ -975,6 +975,10 @@ const (
 type BuildConfigStatus struct {
 	// lastVersion is used to inform about number of last triggered build.
 	LastVersion int64 `json:"lastVersion" protobuf:"varint,1,opt,name=lastVersion"`
+
+	// imageChangeTriggersState is used to capture the runtime state of any ImageChangeTrigers,
+	// including the lastTriggeredImageID and paused fields.
+	ImageChangeTriggersState []ImageChangeTrigger `json:"imageChangeTriggersState,omitempty" protobuf:"bytes,2,rep,name=imageChangeTriggersState"`
```

Comment:
so will the .from value of ImageChangeTrigger have the actual sha value of the specific image that triggered the build?

Reply:
no I think I put the deprecation notice in the wrong spot @bparees ... I got a bit turn around since we are reusing the same struct in spec and status 

I'll work on that, along with trying to get this verify job to pass, and unhold this when things are ready

/hold

Reply:
or conversely, leave the deprecation notice here, and use a new / separate struct with an analogous field

Reply:
OK I think I've sorted this out ... updates pushed